src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -39,6 +42,11 @@ private Conveyor() {
         colorMatch.addColorMatch(NONE);
         cargoPositions.add(DriverStation.Alliance.Invalid.name());
         cargoPositions.add(DriverStation.Alliance.Invalid.name());
+        if (Utils.isBitSet(Robot.pneumaticsBase.getSolenoids(), SOLENOID)) {

```

Comment:
`getSolenoids()` is used in `Solenoid`'s `get()` ([source](https://github.com/wpilibsuite/allwpilib/blob/5635f33a328144542e4ca7947c8c54614ce30de3/wpilibc/src/main/native/cpp/Solenoid.cpp#L50-L53)):
```cpp
bool Solenoid::Get() const {
  int currentAll = m_module->GetSolenoids();
  return (currentAll & m_mask) != 0;
}
```
So I think that unfortunately, that doesn't help either (or if it does, you can simply call `Solenoid`s `get()`). What does `get()` return for a disconnected solenoid?

Also, just a quick question: Aren't double solenoids used in the robot?

Reply:
1. Honestly I am not sure anymore
2. We use single solenoids
3. Yes we will add bits for each subsystem, how would you handle a TODO list?